consensus-types/types/block.go

```diff
@@ -41,14 +41,21 @@ type BeaconBlock struct {
 	ParentRoot common.Root `json:"parent_root"`
 	// StateRoot is the hash of the state at the block.
 	StateRoot common.Root `json:"state_root"`
-	// Body is the body of the BeaconBlock, containing the block's
-	// operations.
+	// Body is the body of the BeaconBlock, containing the block's operations.
 	Body *BeaconBlockBody `json:"body"`
+
+	// BbVersion is the BbVersion of the beacon block.
+	// BbVersion must be not serialized but it's exported
+	// to allow unit tests using reflect on beacon block.
+	BbVersion uint32
 }
 
 // Empty creates an empty beacon block.
 func (*BeaconBlock) Empty() *BeaconBlock {
-	return &BeaconBlock{}
+	return &BeaconBlock{
```

Comment:
I don't want to modify the signature here to pass the `forkVersion` although I think it should be safe